ansible/roles/linux-data-volumes/files/data-volumes-configure-nvme.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Set LVM configuration
+  set_fact:
+    lvm_groups:
+      - vgname: lvm-nvme
+        disks: "{{ openstack_metadata.lvm_disk_nvme | split(',') | map('trim') | list }}"
```

Comment:
When we can't find all the disks on this list on this host, we should skip doing this mount. Equally, we need to double check that the disks are not currently in use, I think. (maybe using ansible devices fact?)